backend/src/routes/blog.ts

```diff
@@ -378,4 +378,23 @@ blogRouter.post("/chat", async (c) => {
     c.status(403);
     return c.json({ error: "Something went wrong", stackTrace: ex instanceof Error ? ex.toString() : JSON.stringify(ex) });
   }
-});
\ No newline at end of file
+});
+blogRouter.get("/blogCount/:id",async(c)=>{
+  try {
+    const id = c.req.param("id");
+    const prisma = getDBInstance(c);
+    const count = await prisma.post.count({
+      where:{
+        authorId:id
+      }
+    });
+    return c.json({
+      count:count
+    });
+  } catch (error) {
+      c.status(400);
+      return c.json({
+        error:"Something Went Wrong"
+      })
+  }
+})
```

Comment:
This API isn't required either. 
The `api/v1/blog/bulkUser/:userId` is called on Profiles page, which returns blogs[]. You can get the blog count by simply doing blogs.length on the frontend.